public/assets/env.js

```diff
@@ -73,5 +73,6 @@ const APP_ENV = {
       rest: 'https://api.devnet.staging.okp4.network:443/rpc',
       ...sharedChainConf
     }
-  ]
+  ],
+  dropDown: { dropDownMaxSearchResults: 10, dropDownTagsDisplayLimit: 4 }
```

Comment:
Although current property names are reasonably clear imho, it is generally desirable to use more descriptive names that correctly reflect their purpose. I would suggest:

```suggestion
  dropDown: { maxDisplayedSearchResults: 10, maxDisplayedTags: 4 }
```